src/main/scala/com/full360/voltdbscala/Client.scala

```diff
@@ -133,12 +131,12 @@ trait Client {
    * @param maxBatchSize Batch size to collect for the table before pushing a bulk insert.
    * @param upsert       set to true if want upsert instead of insert
    * @param f            Function invoked by the BulkLoaderFailureCallBack used for notification of failed inserts.
+   * @param f2           Function invoked by BulkLoaderSuccessCallback used for notifications on successful load operations
    * @return instance of VoltBulkLoader
    * @throws Exception if tableName can't be found in the catalog.
    */
-  def getNewBulkLoader(tableName: String, maxBatchSize: Int, upsert: Boolean = false)(f: (Any, Seq[AnyRef], ClientResponse) ⇒ Unit): VoltBulkLoader =
```

Comment:
I think this method should not be removed. It exist in two variants with method overloading.